### PR TITLE
add the rest of the bridge debug features

### PR DESCRIPTION
**Pull request by bbaldino**

### CODE REVIEW

**Comment by bbaldino** [comment]
jenkins, test this please

File: src/main/java/org/jitsi/videobridge/rest/debug/Debug.java [hunk 1]
```diff
@@ -108,4 +120,29 @@ public String epDebug(@PathParam("confId") String confId, @PathParam("epId") Str
...
+                return ByteBufferPool.getStatsJson().toJSONString();
+            }
+            case QUEUE_STATS: {
+                return videobridgeProvider.get().getQueueStats().toJSONString();
+            }
+            case TRANSIT_STATS: {
+                return PacketTransitStats.getStatsJson().toJSONString();
+            }
+            default: {
+                throw new NotFoundException();
```
Comment (virtuacoplenny) [inline src/main/java/org/jitsi/videobridge/rest/debug/Debug.java]: NotFoundException ends up returning a 404?

File: src/main/java/org/jitsi/videobridge/shim/ChannelShim.java [hunk 2]
```diff
@@ -242,23 +242,13 @@ public void describe(ColibriConferenceIQ.ChannelCommon commonIq)
...
-                    {
-                        ssrcs[i] = (int) source.getSSRC();
-                        i++;
-                    }
-                }
-                if (i < ssrcs.length)
-                {
-                    ssrcs = Arrays.copyOf(ssrcs, i);
-                }
+            if (sources != null) {
```
Comment (virtuacoplenny) [inline src/main/java/org/jitsi/videobridge/shim/ChannelShim.java]: Are these changes related to the debug features?

File: src/main/java/org/jitsi/videobridge/rest/debug/Debug.java (hunk 1 above)
Comment (bbaldino) [reply src/main/java/org/jitsi/videobridge/rest/debug/Debug.java]: yeah

File: src/main/java/org/jitsi/videobridge/shim/ChannelShim.java (hunk 2 above)
Comment (bbaldino) [reply src/main/java/org/jitsi/videobridge/shim/ChannelShim.java]: they are not, consider it a [rider](https://en.wikipedia.org/wiki/Rider_(legislation))...
